legal-api/src/legal_api/reports/business_document.py

```diff
@@ -499,6 +501,54 @@ def _set_liquidation_details(self, business: dict):
                 liquidation_info['recordsOffice'] = records_office_info
         business['liquidation'] = liquidation_info
 
+    def _set_continuation_in_details(self, business: dict):
+        """Set continuation in filing data."""
+        continuation_in_info = {}
+        continuation_in_filing = Filing.get_filings_by_types(self._business.id, ['continuationIn'])
+        if continuation_in_filing:
+            continuation_in_filing = continuation_in_filing[0]
+            continuation_in_data = continuation_in_filing.filing_json['filing']['continuationIn']
+
+            continuation_in_info['filingDateTime'] = continuation_in_filing.filing_date.isoformat()
+            continuation_in_info['mode'] = continuation_in_data.get('mode', 'N/A')
```

Comment:
cannot dependent on mode in the filing_json as its not an expected value as per schema

Reply:
can check if this field has value https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/jurisdiction.py#L34